server/events/project_command_runner.go

```diff
@@ -258,9 +260,7 @@ func (p *DefaultProjectCommandRunner) Plan(ctx command.ProjectContext) command.P
 		ProjectName:       ctx.ProjectName,
 		SilencePRComments: ctx.SilencePRComments,
 	}
-}
-
-// PolicyCheck evaluates policies defined with Rego for the project described by ctx.
+} // PolicyCheck evaluates policies defined with Rego for the project described by ctx.
```

Comment:
Something strange happened, the comment for the following function seems to be on the closing brace of the previous function